src/components/JarSelectorModal.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useMemo } from 'react'
+import * as rb from 'react-bootstrap'
+import { useTranslation } from 'react-i18next'
+import styles from './JarSelectorModal.module.css'
+import Sprite from './Sprite'
+import { calculateFillLevel, SelectableJar } from './jars/Jar'
+
+export default function JarSelectorModal({
+  isShown,
+  title,
+  accountBalances,
+  totalBalance,
+  disabledJar = undefined,
```

Comment:
NIT, as it is not used here and can be changed easily on demand.
Should be an array and named `disabledJars`  – "zero, one, many". 
Disabling zero or one is an edge case derived from the current task.

Reply:
> Disabling zero or one is an edge case

Isn't disabling zero or one the only actual use case we have right now?

Reply:
> > Disabling zero or one is an edge case
> 
> Isn't disabling zero or one the only actual use case we have right now?

Yes, that's why it is a NIT : D
But it is an edge case because we select the destination jar.. would the component be used for selecting the source jar, it might be that all empty jars should be disabled.. 
It is meant as a general purpose component, right?
That's what I meant when I said this is "derived from the current task" at hand.

Reply:
Ah ok now I see what you mean. 👍 

Yeah make sense. But let's cross that bridge when we get to it.